pkg/align/align_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2015-2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package align
```

Comment:
There should be an empty line between the copyright statement and the package statement, otherwise the copyright statement looks like the package comment. And there can only be one of those per directory.